uber/templates/preregistration/preregbase.html

```diff
@@ -69,7 +71,7 @@
                 extra: 0
             }]
         };
```

Comment:
I can go back and check, but to my recollection, most of the people who got two shirts were people who kicked in for the supporter package.  That will still work, this change just prevents someone from kicking in at exactly the "Add a Tshirt" level.

Here's my thinking on why this is a good change:
- If we leave that in as a donation tier for staffers, many people will assume that they need to pay an extra $20 to get a tshirt.
- If we take it out as a donation tier for staffers, some small number of staffers who understand they already get one shirt will be unable to use this form to pre-order a second shirt.

To my mind, situation 2 is preferable to situation 1, especially since they can buy a shirt on-site from the merch booth (though obviously preordering is preferable since then you're guaranteed to have the shirt available in your size when you show up to the merch booth).

Does that all sound reasonable?